app/services/intl.js

```diff
@@ -4,9 +4,9 @@
  */
 
 import Ember from 'ember';
-import Locale from 'ember-intl/models/locale';
-import createFormatCache from 'ember-intl/format-cache/memoizer';
 import { IntlRelativeFormat, IntlMessageFormat } from 'ember-intl/utils/data';
+import createFormatCache from 'ember-intl/format-cache/memoizer';
```

Comment:
That was the issue, I can revisit and figure out a solution for this.

The issue with browserify was not being able to import both mf and rf.  That should be fixed now so soon I plan to remove to global references to these.